packages/dds/shared-object-base/src/sharedObject.ts

```diff
@@ -145,7 +147,12 @@ export abstract class SharedObjectCore<
 
 		assert(!id.includes("/"), 0x304 /* Id cannot contain slashes */);
 
-		this.handle = new SharedObjectHandle(this, id, runtime.IFluidHandleContext);
+		this.handle = new SharedObjectHandle(
+			this,
+			id,
+			// eslint-disable-next-line import/no-deprecated
+			runtime as IFluidDataStoreRuntimeExperimental,
```

Comment:
is the cast even necessary? everything on IFluidDataStoreRuntimeExperimental is optional so should be implicitly convertible from/to IFluidDataStoreRuntime